pkg/storage/swift/swift.go

```diff
@@ -1,33 +1,122 @@
 package swift
 
 import (
+	"fmt"
+	"math/rand"
+	"reflect"
+
 	corev1 "k8s.io/api/core/v1"
 
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/containers"
```

Comment:
Nit - imports should be grouped as follows (separated by empty lines):
1. golang std lib
2. 3rd party
3. k8s.io
4. openshift

We don't always adhere to this, as IDEs can't really enforce this.

Reply:
done